alma/endpoints/merchants.py

```diff
@@ -13,3 +15,16 @@ def me(self, extended=False):
             .get()
             .expectJson(Merchant)
         )
+
+    def fee_plans(self, kind: FeePlanKind, only: Optional[List[int]] = None):
+        params = {"kind": kind.value}
+
+        if only is not None:
+            params["only"] = ",".join(str(i) for i in only)
+
+        return (
+            self.request(f"{self.ME_PATH}/fee-plans")
+            .set_query_params(params)
+            .get()
+            .expect(lambda resp: [FeePlan(fp) for fp in resp.json])
```

Comment:
I start to see this on multiple endpoints, maybe we should take the opportunity to create a `expectFlatList` helper.